spec/services/itsi_authoring/editor_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "spec_helper"
+
+describe ITSIAuthoring::Editor do
+  let(:author)        { FactoryGirl.create(:author) }
+  let(:activity)      {
+    activity = FactoryGirl.create(:activity)
+    activity.user = author
+    activity.save
+    activity.theme = FactoryGirl.create(:theme)
+    activity
+  }
+
+  let(:page)                { FactoryGirl.create(:interactive_page,
+                                                 name: "page 1",
+                                                 position: 0
+                                                 )}
+  let(:library_interactive) { FactoryGirl.create(:library_interactive,
+                                                 name: 'Test Library Interactive',
+                                                 base_url: 'https://concord.org/'
+                                                )}
+  let(:managed_interactive) { FactoryGirl.create(:managed_interactive,
+                                                 library_interactive: library_interactive,
+                                                 url_fragment: "test"
+                                                )}
+
+  before(:each) do
+    page.add_interactive(managed_interactive)
```

Comment:
Since you've gone this far, you should make a second managed_interactive, and add it to the page as an embeddable. I think that is just using the page.add_embeddable method.
That way this test will also cover liine 61 of editor.rb: 
https://github.com/concord-consortium/lara/pull/606/files#diff-f875b83b4abbc964eb099221f462f109R61

Reply:
OK, I think that's all set.